src/en/controllers-backup.md

```diff
@@ -224,9 +224,76 @@ therefore only becomes necessary when *all* controllers have failed. However,
 if a restore *is* applied to a cluster with active members all reachable
 controllers will naturally have their data overwritten.
 
-Section [Recovering from controller failure][recovering-ha-failure] details how
-to deal with a partially degraded cluster. In the advent that all controllers
-are unresponsive the following steps should be taken:
+### Restoring to a cluster
+
+It is not possible at this time to restore while HA is enabled.
+
+To restore to an HA cluster one needs to first remove HA (by removing all but
+one of the controller machines) and then perform a restore operation.
+Presumably HA will be re-enabled afterwards.
```

Comment:
I know what you mean, but I think the use of *presumably* diminishes the authority of the docs.

Reply:
I will put. *HA may then be re-enabled afterwards.*